src/reformat_libyuv.c

```diff
@@ -221,7 +249,8 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
                 return AVIF_RESULT_REFORMAT_FAILED;
             }
             return AVIF_RESULT_OK;
-        } else if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV422) {
+        }
+        if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV422) {
```

Comment:
I rewrote these else-if statements as if statements, so that it will be easier to nest with `#if LIBYUV_VERSION >= 1780`.